vocabularies/gsq-dataset-themes.ttl

```diff
@@ -0,0 +1,100 @@
+    skos:topConceptOf :conceptScheme .
+
+:geoscience-project-data a skos:Concept ;
+    skos:definition "Data, metadata, and media describing project information such as the approval documentation, project planning, and activities performed to generate the primary geoscience data."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Geoscience project data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:other a skos:Concept ;
+    skos:definition "Any other metadata, data, or information that is not described by any of the other GSQ Dataset Themes."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Other"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:operational-data a skos:Concept ;
+    skos:definition "Data describing the operations of mining, drilling, processing, and sales operations e.g. borehole drilling, pipeline operations, processing facility operations."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Operational data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:physical-samples a skos:Concept ;
+    skos:definition "Data and metadata pertaining to physical samples taken during geoscience data acquisition e.g. drill core, rock chips, rock samples from outcrop, soil, sediment, water, drilling fluids, hydrocarbons etc."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Physical samples"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:publications a skos:Concept ;
+    skos:definition "Publications that describe the geology, resource potential, or other areas of departmental interest such as books, journals, periodicals, presentations, posters, academic publications, or multimedia products."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Publications"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:reports a skos:Concept ;
+    skos:definition "Reports lodged as part of an approved study or activity, commonly for the purpose of compliance with permit grant and retention conditions. These reports may be permit based such as resource authority annual reports, activity based such as well completion reports, or non-tenure reports such as departmental survey reports or academic studies."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Reports"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:resource-statistics a skos:Concept ;
+    skos:definition "Statistical data and reports submitted to the department or generated by the department, particularly production, sales and reserve quantities."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Resource statistics"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:spatial-data a skos:Concept ;
+    skos:definition "Data and metadata describing spatial properties and features, including maps, aerial and satellite imagery, and files for use in GIS software."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Spatial data"@en ;
+    skos:topConceptOf :conceptScheme .

```

Comment:
As discussed throughout the data modelling and scoping of system requirements  all geoscience data is spatial in some sense as is all the supporting information that goes with it. Therefore is Spatial Data a redundant category? Virtually all our information would have a spatial aspect.

Reply:
I think there is a difference between a spatial dataset and a dataset that has spatial attributes. Where would you put a topographic map?

Reply:
Topo maps would be under geoscience data. But the same rationale would have to apply to aerial and satellite imagery... which may or may not apply to geoscience in anyway. I'd be dubious labelling it as such.
Hypothetically if we had some environmental data that was not strictly geoscience data it wouldn't fit neatly into any of the categories either. So perhaps there is room for another category for the scientific data that is _not_ strictly geoscience data.

@mark-gordon-dnrme thoughts?

Reply:
The definition of Spatial Data as "Data and metadata describing spatial properties and features, including maps, aerial and satellite imagery, and files for use in GIS software." might be contributing to this ambiguity around the content.

The relevant items I can think of that would fit this current definition would be:

- Wellcards (lat/lon, depth, geometry), las files & other activity reports (Annual etc.)
- Digital Elevation Models
- Topographic Maps
- Tenure plans
- Mine Plans
- Radar traces
- Satellite Imagery
- Aerial Photography
- LIDAR
- JORC reports
- Journal articles
- Geophysics
etc.

What is the overall purpose for this category, what are we trying to capture in it?

Reply:
Change spatial data definition to:
> Spatial data including maps, aerial and satellite imagery, and files for use in GIS software.